_posts/2018-04-28-hackillinois-2018-recap.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Dreamt it. Built it. HackIllinois 2018, A Recap."
+date: 2018-04-28 00:00:00
+categories: recap
+tags: HackIllinois2018
+image: /assets/article_images/hackillinois_2018.jpg
+author: Justin Yang
+---
+
+This February, HackIllinois welcomed over a thousand students to the University of Illinois at Urbana-Champaign for an excitement-packed weekend to develop open-source projects.
+
+HackIllinois 2018 kicked off Friday afternoon, with students lining up for registration and to chat with sponsors at the company expo. After everyone filled up on dinner from Panda Express and Kohinoor, they heard opening remarks from the Opening Ceremony. Attendees then broke out into two panels based on their interests: Web & Data and Systems where open-source mentors spoke about their projects and the open-source community in general.
+
+Attendees, however, weren't just at HackIllinois to hear about open source. As the panels concluded, attendees formed into groups via the project matching sessions to work on open-source projects -- be it with open-source projects to improve popular open-source projects such as Rust and Freenode.
```

Comment:
Reduce the amount of times saying "open-source" in this paragraph. Also the "be it" part sounds a little confusing grammatically.